### ISSUE TITLE
Stock price predictor

### ISSUE THREAD

**Issue by garimasingh128** [garimasingh128/awesome-python-projects#46]
Stock price predictor is a machine learning system that learns about the current performance of a company and based on this, predicts future stock prices. To begin working with stock market data, you can predict and make a simple machine learning problem like predicting 6-month price movements based on fundamental indicators or build time series models, or even recurrent neural networks, on the delta between implied and actual volatility from an organizations’ quarterly report. One can also extend this project to find similar stocks based on their price movements and other factors and look for periods when their prices diverge.

**Comment by ayushsurana710** [comment]
Can you please assign this issue to me?

**Comment by ayushsurana710** [comment]
Hello Garima ,
Do I have to show stock price prediction for a particular stock or does the
project expect something else from my side. I request to clarify this.

Thanks and Regards,
Ayush Surana
ᐧ

**Comment by garimasingh128** [comment]
Yes. I think of a single stock from a single company as a start. Then you can extend its functionalities as you wish to.

**Comment by ayushsurana710** [comment]
Thank you!
ᐧ

**Comment by ksdkamesh99** [comment]
I am interested to work on this project @garimasingh128 Please Assign this to me

**Comment by ksdkamesh99** [comment]
Thanks @garimasingh128  for assigning me
